assignments/function-conversion.js

```diff
@@ -1,30 +1,37 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
+// let myFunction = function () { 
 // console.log("Function was invoked!");
 // };
 // myFunction();
 
+let myFunction = function1 => console.log("Function was invoked!"); 
+
```

Comment:
To test your function is logging the result you intended, make sure to invoke the function. `myFunction()`